nano/nano_node/daemon.cpp

```diff
@@ -154,6 +155,8 @@ void nano_daemon::daemon::run (boost::filesystem::path const & data_path, nano::
 					std::cout << "Voting with more than one representative can limit performance: " << voting << " representatives are configured" << std::endl;
 				}
 				node->start ();
+				boost::interprocess::named_mutex process_mutex (boost::interprocess::open_or_create, "nano_node_process");
+				process_mutex.try_lock ();
```

Comment:
This isn't checking the result of try_lock.

Reply:
Yeah it prevents from executing confirmation_height_clear command when other node process is running, not the other way.
Do you suggest to check the result of try_lock and exiting the node process if false?
I think it's too risky to check the mutex on node side due to possibility of mutex not being unlocked from previous run, for example when node process was killed. Then you wouldn't be able to run the node until you reboot the machine.